strategy.js

```diff
@@ -3,11 +3,11 @@ module.exports = function (passport, util) {
 
   function MockStrategy (options, verify) {
     if (!verify) { throw new TypeError('MockStrategy requires a verify callback'); }
-    if (!options.callbackURL) { throw new TypeError('MockStrategy requires a callbackURL'); }
+    if (!options.callbackURL && !options.client.redirect_uris[0]) { throw new TypeError('MockStrategy requires a callbackURL'); }
```

Comment:
Will change, and there is a test for the error thrown: https://github.com/shake-apps/passport-mocked/pull/14/files#diff-ecccc4504df76eb68f5239a9f48d7d4bL31